docs/Operators.md

```diff
@@ -7135,6 +7136,153 @@ Other versions of this operator: <a href="Changelog.md#If-1">If-1</a>
 </dl>
 
 
+### <a name="ImageToCol"></a><a name="imagetocol">**ImageToCol**</a>
```

Comment:
With this new op, can "SpaceToDepth" be replaced/deprecated? looks to me this one is more general.

To replace "SpaceToDepth", a little bit more needs to be updated, as "SpaceToDepth" is not squeezing the last 2 dimensions into one (while this op does). Maybe adding all parameters used to define the block into "SpaceToDepth" is better?

Reply:
Thanks @linkerzhang 
I talked about this with Spandan and Emad as well. So, as you mentioned, this operator is a the more general version of SapceToDepth (except the mode attribute is not supported). I think we can modify this operator's name and functionality further (maybe not flattening the last dimension, support for CRD/DCR re-arrangemen modes, etc.), and change SapceToDepth to a function that uses this operator internally.
However,  SpaceToDepth and DepthToSpace are currently complimentary ops, and I think it's better to keep them as such. We can add an operator equivalent for ColToImage (with a more general naming), and change both SpaceToDepth and DepthToSpace operators as functions.

Reply:
do you have plan to do the move? (move SpaceToDepth to function).

I am also OK to deprecate SpaceToDepth if this OP can be changed to not merge the last two dimension (split them in to sub dimensions).

ONNX op has versioning, it's definitely OK to deprecate one as long as there's new one which can do the job @ebarsoum / @postrational  may comment more on this.

Reply:
Based on our discussion, this work will move to 1.8. And we will update the existing SpaceToDepth and DepthToSpace to be more general.

Reply:
great! confirm: so the plan is to update SpaceToDepth/DepthToSpace more general and move this one to function?

Reply:
Should we change the Milestone to 1.8?

Reply:
@postrational I think 1.7 will take this one